src/main/java/org/jabref/gui/preferences/ImportTab.fxml

```diff
@@ -28,8 +27,7 @@
                   prefWidth="300" minWidth="300" maxWidth="300"/>
 
         <Label text="%File directory pattern" GridPane.rowIndex="1"/>
-        <TextField fx:id="fileDirPattern" GridPane.columnIndex="1" GridPane.rowIndex="1"
+        <TextField fx:id="fileDirectoryPattern" GridPane.columnIndex="1" GridPane.rowIndex="1"
                    prefWidth="300" minWidth="300" maxWidth="300"/>
-        <CheckBox fx:id="downloadLinkedFiles" text="%Download linked online files" GridPane.rowIndex="2"/>
```

Comment:
Is there a reason you removed the "Download linked online files"`?

Reply:
Yes, there is no need for this checkbox in the preferences if its also displayed in the dialog anyway.